### PR TITLE
Weapons haks

### PR DESCRIPTION
**Pull request by Xisifer**
New weapons:
8 Blaster Pistols
13 Blaster Rifles 
1 Power Hammer (greataxe)

KNOWN ISSUE: Weapon icons do not have transparency.

### CODE REVIEW

**Comment by Xisifer** [comment]
Adjusted position and rotation for cantina band instruments, by request of @announcer348.

File: swlor2_mdl/waxgr_m_081.mdl [hunk 1]
```diff
@@ -0,0 +1,1039 @@
+# Exported from NeverBlender 2.8.33 - Friday, 2021-12-17
```
Comment (zunath) [inline swlor2_mdl/waxgr_m_081.mdl:1]: Please compile this model and any others which aren't already.

**Comment by zunath** [comment]
I'm holding off on merging this until the icons have been fixed. I think this process is pretty straightforward.

1. Open the TGA in an editor. I used Paint.NET but I'm positive Photoshop can do this too.
2. Select the whole background color 
3. Press delete
4. Save the file

Before:
image

Selection:
image

After:
image

**Review by zunath — CHANGES_REQUESTED** [review]
Looks awesome but please address the icon backgrounds and compile the models. Let me know if you need help!

**Review by zunath — APPROVED** [review]

**Pull request merged**